tests/training/metrics_test.py

```diff
@@ -0,0 +1,139 @@
+# pylint: disable=no-self-use,invalid-name
+import torch
+import pytest
+import numpy
+
+from allennlp.common.testing import AllenNlpTestCase
+from allennlp.common.checks import ConfigurationError
+from allennlp.training.metrics import CategoricalAccuracy, F1Measure
+
+class MetricsTest(AllenNlpTestCase):
+
+    def test_categorical_accuracy(self):
```

Comment:
Might want to have a test here for multiple calls to the metric, and the reset functionality.